src/csharp/GRPC-TOOLS-MSBUILD.md

```diff
@@ -0,0 +1,215 @@
+# Grpc.Tools MSBuild integration overview
+
+This is an overview for maintainers of Grpc.Tools.
+
+The Grpc.Tools NuGet package provides custom build targets to make it easier to specify ._proto_ files
```

Comment:
let's use backticks for references to paths, code, filenames etc.
e.g. replace by `.proto`. Let's do this throughout this file. This will also help you with the escaping  you currently need to do with the `_` formatting (when file names need escaping, it's very easy to make mistakes).